Project1/application.py

```diff
@@ -110,5 +110,29 @@ def admin() :
     users = User.query.order_by(User.timestamp.desc()).all()
     return render_template("admin.html", users=users)
 
+@app.route("/bookpage")
+def bookpage(isbn):
+    # Get data about book from database
+        result = db.execute("SELECT isbn, title, author, year FROM books WHERE id = :id",
```

Comment:
As sql queries change for different databases, use db.query

Reply:
Mam, please check "application-bookpage" commit. You have reviewed the outdated one. I have updated the changes as  "application-bookpage" commit.Please review it. I have used db.query only.

Reply:
ok